src/ipsis/course-merge/BaseCollectionEntity.js

```diff
@@ -1,6 +1,22 @@
 import { Entity } from '../../es6/Entity.js';
 
 export class BaseCollectionEntity extends Entity {
+	constructor(entity, token, listener) {
+		super(entity, token, listener);
+		this._allEntities = null;
+	}
+
+	courseOfferings() {
```

Comment:
Should we call this `courseMergeOfferings` to avoid being a breaking change for `CourseMergeOfferingCollectionEntity`?

Reply:
Done